doc/API.md

```diff
@@ -44,6 +44,7 @@ The ```options``` parameter is JS object with the following properties:
     - `enableWindowOnErrorHandler` - boolean property (default false). Enables/disables attaching global onerror handler (window.onerror).
     - `disableThirdPartyRequests` - if true - callstats will be disabled and the callstats API won't be included.
     - `enableAnalyticsLogging` - boolean property (default false). Enables/disables analytics logging.
+    - `externalStorage` - Obejct that implements the Storage interface. If specified this object will be used for storing data instead of `localStorage`.
```

Comment:
```suggestion
    - `externalStorage` - Object that implements the Storage interface. If specified this object will be used for storing data instead of `localStorage`.
```